services/order.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+@transaction.atomic
+def create_order(tickets: list[dict],
```

Comment:
Using `list[dict]` as a default argument is not recommended because lists are mutable. Consider using `Optional[List[Dict]]` from the `typing` module to avoid potential issues.